restapi.go

```diff
@@ -157,7 +157,7 @@ func (s *Session) RequestWithLockedBucket(method, urlStr, contentType string, b
 		s.log(LogInformational, "Rate Limiting %s, retry in %d", urlStr, rl.RetryAfter)
```

Comment:
Since `rl.RetryAfter` is now a `float64`, this format verb should be `%f`.

```suggestion
		s.log(LogInformational, "Rate Limiting %s, retry in %f", urlStr, rl.RetryAfter)
```